01-node-tutorial/answers/prompter.js

```diff
@@ -21,17 +21,33 @@ const getBody = (req, callback) => {
 };
 
 // here, you could declare one or more variables to store what comes back from the form.
-let item = "Enter something below.";
+let name = "Enter your name.";
+let backgroundColor = "#ffffff";
+let prevName = "Enter your name.";
+let name2 = "";
+let prevName2 = "";
```

Comment:
This is a little nit-picky but variables like `name2` are not ideal. Even something like `secondName` or something is better. I know it is hard but  try your best to abstract your variable names away from needing numbers.

Reply:
Thanks for the review! I have corrected the variable names.